### ISSUE TITLE
Configuration Location

### ISSUE THREAD

**Issue by LiveFreeAndRoam** [rangav/thunder-client-support#320]
I am a new user of Thunder Client v1.9.1. It is a fabulous tool, thank you!!

Where is the Thunder-Client configuration information stored?

I have created quite a few collections with sub-folders, tests, etc. I would like to place these configurations under git control, but I don't know where these files are stored.

Similarly, I would like to share these files, but cannot find an export option.  How can I export them?

I played with "Custom Location" setting.  I noticed the custom location creates ThunderActivity.db, ThunderCollection.db, etc.  These seem to be JSON files, so I wonder if editing them directly is supported?

**Comment by rangav** [comment]
Hi @LiveFreeAndRoam you cannot edit them directly, it will corrupt the data.

**Comment by LiveFreeAndRoam** [comment]
Thanks @rangav.

I have a simple edit to make to many of the requests and it would be a lot easier to perform that edit across all requests using the VS Code editor.

BTW, I found the location of the configuration file on my laptop.  It is:

`C:\Users\<myusername>\AppData\Roaming\Code\User\globalStorage\rangav.vscode-thunder-client`

Under that directory there are several db files (all formatted as JSON): `thunderActivity.db`, `thunderclient.db`, `thunderCollection.db`, `thunderEnvironment.db`, `thunderResponses.db`.

I am now going to copy these files into my project's workspace and then point the `Custom Location` setting at them.  Hopefully this will work. I'll then have them placed under my project's git control.

**Comment by LiveFreeAndRoam** [comment]
I'm all setup now.

I first created a folder in under my worksapce and copied the *.db files to it.  I pointed `Custom Location` setting at it, reloaded VSC and Thunder Client could see my configurations.  BTW, I needed to provide the absolute path, rather than something relative to my project root.  An absolute path is undesirable because the workspace could be moved elsewhere.

I next discovered the `Load From Project` setting, which creates a `thunder-tests` folder under the workspace root. After checking that option and reloading VSC, I still needed to send one request before the `thunder-tests` folder was created.  After that, I overwrote that directory with my files from `%APPDATA%\Code\User\globalStorage\rangav.vscode-thunder-client`, reloaded VSC and now I have it all working from under my workspace.  The `.db` files are now under git control.

I'm past the initiation phase, phew!
